### PR TITLE
events integ tests

### PR DESCRIPTION
**Pull request by jhsinger-klotho**
events integ test and policy pack

also added persist and another get endpoint to test if our pubsub is truly working. Before the pubsub didnt write anything anywhere so tests would always pass if the initial lambda got a response

### CODE REVIEW

File: ts-events/README.md [hunk 1]
```diff
@@ -61,7 +66,16 @@ curl -X POST https://<...>.execute-api.<YOUR_REGION>.amazonaws.com/stage/hello/u
 # >
 ```
 
+<<<<<<< HEAD
```
Comment (ewucc) [inline ts-events/README.md:69]: looks like some merge stuff got left in

File: ts-events/README.md [hunk 2]
```diff
@@ -61,7 +66,16 @@ curl -X POST https://<...>.execute-api.<YOUR_REGION>.amazonaws.com/stage/hello/u
...
 You will see 3 different cloudwatch log groups specifying the sequence of events in the cloud deployed application.
+=======
```
Comment (ewucc) [inline ts-events/README.md:71]: same ^

File: ts-events/README.md [hunk 3]
```diff
@@ -61,7 +66,16 @@ curl -X POST https://<...>.execute-api.<YOUR_REGION>.amazonaws.com/stage/hello/u
...
+```sh
+$ curl https://<...>.execute-api.<YOUR_REGION>.amazonaws.com/stage/hello/user
+# > user%
+```
+
+You will see 3 different cloudwatch log groups specifying how the sequence of events in the cloud deployed application.
+>>>>>>> 0e6baf0 (placeholder)
```
Comment (ewucc) [inline ts-events/README.md:78]: same ^

File: ts-events/test/happy_path.test.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import {expect} from '@jest/globals';
+
+const testUser = "user"
```
Comment (ewucc) [inline ts-events/test/happy_path.test.ts:3]: rip timmy the test user

**Review by ewucc — CHANGES_REQUESTED** [review]

File: ts-events/test/pulumi/index.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    description: "Lambda integration tests.",
+    enforcementLevel: "mandatory",
+    validateStack: async (args, reportViolation) => {
+        const lambdas = args.resources.filter(r => r.isType(aws.lambda.Function));
+        if (lambdas.length !== 3) {
+            reportViolation(`Expected three lambda function but found ${lambdas.length}`);
+            return;
+        }
+
+        const lambda = lambdas[0].asType(aws.lambda.Function)!;
```
Comment (yuval-klotho) [inline ts-events/test/pulumi/index.ts:16]: Should we test all of them (not just the first one)?

**Review by ewucc — APPROVED** [review]
Yuval had an open comment but i think it's non blocking for now

**Pull request merged**